parlai/utils/safety.py

```diff
@@ -35,15 +35,16 @@ def share(self):
         shared = {'model': self.model.share()}
         return shared
 
-    def _create_safety_model(self):
+    def _create_safety_model(self, custom_model_file):
         from parlai.core.params import ParlaiParser
 
         parser = ParlaiParser(False, False)
         TransformerClassifierAgent.add_cmdline_args(parser)
+        model_file = 'zoo:dialogue_safety/single_turn/model'
+        if custom_model_file:
```

Comment:
maybe we should add a check that the custom model file exists?

Reply:
yeah good point. I was hoping to raise error requireModelExists=True but it seems like the default for creating agent is requireModelExists=False.